src/modules/client/components/clientAlerts/ClientAlertCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@mui/material';
+import { ReactNode } from 'react';
+import TitleCard from '@/components/elements/TitleCard';
+import { ClientAlertType } from '@/modules/client/components/clientAlerts/ClientAlert';
+import ClientAlertStack from '@/modules/client/components/clientAlerts/ClientAlertStack';
+
+export enum AlertContext {
+  Client = 'Client',
+  Household = 'Household',
+}
+
+interface ClientAlertCardProps {
+  alertContext: AlertContext;
+  clientAlerts: ClientAlertType[];
+  children?: ReactNode;
+}
+const ClientAlertCard: React.FC<ClientAlertCardProps> = ({
+  alertContext = AlertContext.Client,
+  clientAlerts,
+  children,
+}) => {
+  const title = `${alertContext} Alerts (${clientAlerts.length})`;
+
+  return (
+    <TitleCard
+      title={title}
+      headerVariant='border'
+      headerTypographyVariant='body1'
+    >
+      <Box sx={{ m: 2 }}>
+        {clientAlerts.length === 0 && (
+          <Box
+            sx={{
+              p: 2,
+              backgroundColor: '#fafafa',
+              color: '#00000060',
+              borderRadius: 1,
+              display: 'flex',
+              justifyContent: 'space-between',
+            }}
+          >
```

Comment:
Using a box (div) is totally fine, but I usually use `Stack` for something like this: https://mui.com/material-ui/react-stack/


Here's my suggestion
```
          <Stack
            direction='row'
            justifyContent='space-between'
            alignItems='center'
            sx={{
              p: 2,
              backgroundColor: (theme) => theme.palette.grey[50],
              color: 'text.secondary',
              borderRadius: 1,
            }}
          >
```

The Grey 50 and secondary text come from figma:

<img width="948" alt="Screenshot 2024-02-02 at 9 42 00 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/f86583b3-2c3f-41be-b6e7-e5318c60ca63">

Reply:
Nice :) thank you